docs/en/rst/api/core/v1/component.rst

```diff
@@ -18,6 +18,12 @@ To get information about the General component under the Firefox product:
 
    GET /rest/component/Firefox/General
 
+To get information about a component where the product name contains a slash (/) character.
+
+.. code-block:: text
+
+   GET /rest/component?product=Firefox%20%2F%20Bugs&component=General
```

Comment:
We could make the help text clearer by explicitly stating "named parameters must be used instead of path based parameters".